packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -480,6 +486,27 @@ async function migrateMultiEnv(projectPath: string): Promise<void> {
       configDev["skipAddingSqlUser"] =
         envDefault[ResourcePlugins.AzureSQL][EnvConfigName.SqlSkipAddingUser];
     }
+    if (envDefault[ResourcePlugins.Aad]?.[EnvConfigName.AadSkipProvision] === "true") {
+      Object.assign(configDev, { auth: {} });
+      if (envDefault[ResourcePlugins.Aad][EnvConfigName.OAuthScopeId]) {
+        Object.assign(configDev.auth, {
```

Comment:
can we use `configDev.auth.accessAsUserScopeId = xxx` to align the coding style with other part (e.g. https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/core/middleware/projectMigrator.ts#L858)?